skimage/filters/_identity.py

```diff
@@ -0,0 +1,29 @@
+"""
+Identify function.
+
+Return original image.
+"""
+
+
+def identity(image, **kwargs):
```

Comment:
I am also not a fan of defining this in the public API. For convenience in the example script, I would just add something minimal within the script itself, such as:

```python
def identity(image, **kwargs):
    """Return the original image, ignoring any kwargs."""
    return image
```